docs/proposals/security-context-constraints.md

```diff
@@ -0,0 +1,303 @@
+## Abstract
+
+Security context constraints relate [security contexts](./security_contexts.md) and
+[service accounts](http://docs.k8s.io/design/service_accounts.md).  They are configured for a
+cluster, namespaces, and users within the cluster to govern how a [security context](./security_contexts.md) is established.
```

Comment:
This paragraph is a little confusing, I think.  "They are configured for" is a little too ambiguous.  

To clarify, the SCC is a root scoped object.  Ultimately what seemed to work best is to grant access to the SCC based on groups and users that can be retrieved from `user.Info`.  This allows you to easily control who can use the SCC at any level which is nice.   Right now the granting is done by the `Users` and `Groups` fields on the SCC.  

I will take another pass at the design doc.